scheduler/scheduler.go

```diff
@@ -289,6 +289,14 @@ func generateIntentHost(d distro.Distro) (*host.Host, error) {
 	hostOptions := host.CreateOptions{
 		UserName: evergreen.User,
 	}
+	if d.ContainerPool != "" {
+		settings, err := evergreen.GetConfig()
```

Comment:
I worry that this way of getting configs is expensive, and we're going to redo this work for every host that has a container, and the work would basically always be the same?

Reply:
I had overlooked the fact that this is already in memory so I'm now passing it through